src/store/actions/set_active_item.js

```diff
@@ -25,5 +25,18 @@ export default firestoreAction(async ({ bindFirestoreRef, unbindFirestoreRef, co
   const kpisRef = db.collection('kpis').where('parent', '==', item);
   await bindFirestoreRef('kpis', kpisRef, { maxRefDepth: 0 });
 
+  const products = await db
+        .collection('products')
+        .where('department', '==', item)
+        .limit(10)
+        .get()
+        .then((snapshot) => snapshot.docs.map((doc) => doc.ref));
+
+  const subKpisRef = db
+        .collection('kpis')
+        .where('parent', 'in', products);
```

Comment:
Tror denne bugger hvis man ikke har produkter? (Fører til `404`-visning.)

`FirebaseError: Invalid Query. A non-empty array is required for 'in' filters.`